readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -216,6 +223,30 @@ def __init__(self, *args, **kwargs):
         super(HtmlBuilder, self).__init__(*args, **kwargs)
         self.sphinx_builder = 'readthedocs'
 
+    def move(self, **__):
+        super(HtmlBuilder, self).move()
+        if self.project.has_feature(Feature.BUILD_JSON_ARTIFACTS_WITH_HTML):
+            # Copy json artifacts to its own directory
```

Comment:
As the original SearchBuilder isn't executed, we need a move the assets to the other location, I'm not sure if here is the best place, but I feel like it belongs here (since the json output now comes from the html build)

Reply:
Seems as good a place as any.